qiskit/algorithms/time_evolvers/pvqd/pvqd.py

```diff
@@ -74,17 +74,17 @@ class PVQD(RealTimeEvolver):
             import numpy as np
 
             from qiskit.algorithms.state_fidelities import ComputeUncompute
+            from qiskit.algorithms.time_evolvers import TimeEvolutionProblem
             from qiskit.algorithms.time_evolvers.pvqd import PVQD
```

Comment:
A lot of the newer additions, such as gradients, state-fidelities and the new eigensolvers are all only importable via `qiskit.algorithms.<submodule>` so to be consistent in the snippet I thought it would be nice to use the one extra specifier and import from `qiskit.algorithms.time_evolvers` 🙂

Reply:
I am generally against longer input paths, but in this case I agree with Julien, it looks more consistent.

Reply:
Well its only the new (min)eigensolvers and the TrotterQRTE as the name clashes on the algorithms meant they cannot be imported at the algorithms level yet until the older ones are eventually removed. I think gradients and statefidelties are more like optimizers which were/are imported from their modules. The main algorithms were always importable and documented at the algorithms level. Thing I have seen people go into the docs looking for the stuff via the imports and cannot then find them if the import and docs mismatch. Eg this is where you find PVQD in docs, ie., not in time_evolvers https://qiskit.org/documentation/stubs/qiskit.algorithms.PVQD.html#qiskit.algorithms.PVQD